forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -566,7 +567,8 @@ private Card changeZone(final Zone zoneFrom, Zone zoneTo, final Card c, Integer
         // 400.7g try adding keyword back into card if it doesn't already have it
         if (zoneTo.is(ZoneType.Stack) && cause != null && cause.isSpell() && !cause.isIntrinsic() && c.equals(cause.getHostCard())) {
             if (cause.getKeyword() != null && !copied.getKeywords().contains(cause.getKeyword())) {
-                copied.addChangedCardKeywordsInternal(ImmutableList.of(cause.getKeyword()), null, false, game.getNextTimestamp(), null, true);
+                KeywordInterface kw = cause.getKeyword();
```

Comment:
Ideally we avoid doing this a second time 🤔
or does it get dropped from the `checkStaticAbilities` above...?